### PR TITLE
Updated Searchable docs with geo query instructions

### PR DESCRIPTION
**Pull request by Rafcin**
_Issue #, if available:_

The docs used to have info on how to set up `geo_point` queries with Elasticsearch.

_Description of changes:_

I added new documentation under searchable on how to create a custom resolver using the new method.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

File: src/pages/cli/graphql/search-and-result-aggregations.mdx [hunk 1]
```diff
@@ -432,4 +432,165 @@ Type definition of the `@searchable` directive:
 # Streams data from DynamoDB to OpenSearch and exposes search capabilities.
 directive @searchable(queries: SearchableQueryMap) on OBJECT
 input SearchableQueryMap { search: String }
+```
+
+## Geo Queries
+Clients using OpenSearch or Elasticsearch may want to leverage the geo features to search locations or boundaries. OpenSearch and Elasticsearch both offer `geo_point` and `geo_shape` types to help clients perform these types of queries.
+
+Creating a geo query is straightforward. The first step is to create a custom resolver. For more information on creating custom resolvers check [here](https://docs.amplify.aws/cli/graphql/custom-business-logic/#how-it-works-1).
```
Comment (josefaidt) [inline src/pages/cli/graphql/search-and-result-aggregations.mdx:440]:
```suggestion
Creating a geo query is straightforward. The first step is to [create a custom resolver](https://docs.amplify.aws/cli/graphql/custom-business-logic/#vtl-resolver).
```

File: src/pages/cli/graphql/search-and-result-aggregations.mdx [hunk 2]
```diff
@@ -432,4 +432,165 @@ Type definition of the `@searchable` directive:
...
+        path.join(__dirname, '..', 'Query.nameOfQuery.res.vtl')
+      ).renderTemplate(),
+      // If you are using Elasticsearch this will be ElasticSearchDataSource, and if you are using OpenSearch this will be OpenSearchDataSource
+      dataSourceName: "OpenSearchDataSource" // DataSource name
+    })
+  }
+}
+```
+
+Before you create the templates, make sure to create a location field on your model, and properly map the `geo_point` type to the location field.
```
Comment (josefaidt) [inline src/pages/cli/graphql/search-and-result-aggregations.mdx:509]:
```suggestion
Before you create the templates make sure to create a location field on your model, and properly map the `geo_point` type to the location field. 
```

File: src/pages/cli/graphql/search-and-result-aggregations.mdx [hunk 3]
```diff
@@ -432,4 +432,165 @@ Type definition of the `@searchable` directive:
...
+    total: Int
+    nextToken: String
+  ): ModelNameConnection
+}
+
+```
+
+This should be done before data is streamed or if you have data use reindex.
+
+```ts
```
Comment (josefaidt) [inline src/pages/cli/graphql/search-and-result-aggregations.mdx:535]:
```suggestion
```text
```

File: src/pages/cli/graphql/search-and-result-aggregations.mdx [hunk 4]
```diff
@@ -432,4 +432,165 @@ Type definition of the `@searchable` directive:
...
+        "type": "geo_point"
+      }
+    }
+  }
+}
+```
+
+Next, create the VTL request and response templates with `geo_point`. 
+
+```ts
```
Comment (josefaidt) [inline src/pages/cli/graphql/search-and-result-aggregations.mdx:550]:
```suggestion
```vtl
```

File: src/pages/cli/graphql/search-and-result-aggregations.mdx [hunk 5]
```diff
@@ -432,4 +432,165 @@ Type definition of the `@searchable` directive:
...
+"items": $items,
+  "total": $ctx.result.hits.total.value,
+  "nextToken": $nextToken
+})
+```
+
+The following query returns all results within a 200km range of lat 40 and lon -70.
+> **Note:** In versions of OpenSearch >= 1.1, you no longer need to use /doc in the index path. Doing so will result in errors when calling the query.
+
+```ts
```
Comment (josefaidt) [inline src/pages/cli/graphql/search-and-result-aggregations.mdx:568]:
```suggestion
```vtl
```

**Review by josefaidt — CHANGES_REQUESTED** [review]
Hey @Rafcin :wave: thank you for contributing! I think this is great to include, and have requested a few small changes 🙂

**Review by josefaidt — APPROVED** [review]

**Comment by josefaidt** [comment]
cc @renebrandel for visibility/thoughts

**Comment by josefaidt** [comment]
bump @renebrandel and @swaminator for visibility. perhaps @AaronZyLee for searchable input

**Comment by hbuchel** [comment]
Closing this PR due to its age and the docs site has since been refactored. Please feel free to update with latest and re-open if you feel it is still relevant, thanks.

**Pull request closed without merging**